dev/io.openliberty.transport.ssl_fat/fat/src/io/openliberty/transport/ssl/FATSuite.java

```diff
@@ -0,0 +1,38 @@
+/*******************************************************************************
+ * Copyright (c) 2017, 2024 IBM Corporation and others.
```

Comment:
Should we have an update to 2025 here since we're adding a repeat or is this exactly as it was in ws-cd-open?